### PR TITLE
feat(groups): Enforce MFA [DEVOP-1725]

### PR DESCRIPTION
**Pull request by jai**
## Pull Request Submission Checklist

Please confirm that you have done the following before requesting reviews:

- [x] I have confirmed that the PR type is appropriate for the change I am making according to the [Honest Pull Request and Commit Message Naming Conventions](https://www.notion.so/honestbank/Pull-Request-and-Commit-Message-Naming-Conventions-bd97f2cbb34c4c73b1ff3a3e384b850c).
- [x] I have typed an adequate description that explains **why** I am making this change.
- [x] I have installed and run standard pre-commit hooks that lints and validates my code.

### Description

Enforce MFA on groups.

- - -
This change is [Reviewable](https://reviewable.io/reviews/honestbank/terraform-aws-iam/14)

### CODE REVIEW

**Comment by jai** [comment]
Current dependencies on/for this PR:
* main
  * **PR #14** <a href="https://app.graphite.dev/github/pr/honestbank/terraform-aws-iam/14" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/honestbank/terraform-aws-iam/14?utm_source=stack-comment).

**Review by poom — APPROVED** [review]

File: modules/group/main.tf [hunk 1]
```diff
@@ -23,3 +23,26 @@ resource "aws_iam_group_policy_attachment" "policy_attachments" {
     aws_iam_group.group,
   ]
 }
+
+resource aws_iam_group_policy mfa {
+
```
Comment (MXfive) [inline modules/group/main.tf:28]:
```suggestion
```

File: modules/group/main.tf [hunk 2]
```diff
@@ -23,3 +23,26 @@ resource "aws_iam_group_policy_attachment" "policy_attachments" {
     aws_iam_group.group,
   ]
 }
+
+resource aws_iam_group_policy mfa {
```
Comment (MXfive) [inline modules/group/main.tf:27]: @jai Why are there no quotes around the resource type and name?

**Review by MXfive — DISMISSED** [review]

File: modules/group/main.tf (hunk 2 above)
Comment (jai) [reply modules/group/main.tf]: fixed

File: modules/group/main.tf (hunk 1 above)
Comment (jai) [reply modules/group/main.tf]: fixed

**Review by wallsfantasy — APPROVED** [review]
LGTM

**Pull request merged**